invoke/runners.py

```diff
@@ -476,7 +490,7 @@ def start(self, command):
             )
 
     def default_encoding(self):
-        return locale.getpreferredencoding(False)
+        return locale.getpreferredencoding()
```

Comment:
I'm not sure what you mean by "this is not a fix for any platform". My point (as best I can recall it) in using False was simply to avoid messing with global state if it's not necessary.

But on reflection, it seems silly that the default behaviour would be the wrong thing to do. So I'm happy with this part of the change. I was probably over-thinking things in my original change...